forte/dmrg/dmrgscf.cc

```diff
@@ -206,8 +209,8 @@ void DMRGSCF::copyCHEMPS2MXtoPSIMX(CheMPS2::DMRGSCFmatrix* source,
 }
 
 void DMRGSCF::buildQmatOCC(CheMPS2::DMRGSCFmatrix* theQmatOCC, CheMPS2::DMRGSCFindices* iHandler,
-                           psi::SharedMatrix MO_RDM, psi::SharedMatrix MO_JK, psi::SharedMatrix Cmat,
-                           std::shared_ptr<psi::JK> myJK) {
+                           psi::SharedMatrix MO_RDM, psi::SharedMatrix MO_JK,
+                           psi::SharedMatrix Cmat, std::shared_ptr<psi::JK> myJK) {
```

Comment:
While I understand this is beyond the scope of the PR, I think you want references throughout this file, not shared pointers.

Reply:
Oh, yes. We need to leave this for another PR.